README.md

```diff
@@ -82,6 +82,33 @@ A list of configuration options which you need
   - `ALEXANDRIA_THUMBNAIL_HEIGHT`: Height of generated thumbnails
   - `ALEXANDRIA_ENABLE_CHECKSUM`: Set to `false` to disable file checksums. Checksums are calculated after upload to allow later verification (not implemented in Alexandria)
 
+- Storage configuration
+
+  Storage backends are configured globally. The storable object bears information on the encryption status allowing the ORM appropriate handling of the data.
+
+  - `FILE_STORAGE_BACKEND`: Set the backend for file uploads. `django-storages` is available (default: `django.core.files.storage.FileSystemStorage`)
+
+  Encryption:
+  - `ALEXANDRIA_ENABLE_AT_REST_ENCRYPTION`: Set to `true` to enable at-rest encryption of files (does nothing by itself unless `ALEXANDRIA_ENCRYPTRION_METHOD` is set to a supported method)
```

Comment:
I think this is dangerous. Enabling encryption should *force* you to configure everything needed. You shouldn't be able to set `ALEXANDRIA_ENABLE_AT_REST_ENCRYPTION=True` and allow unencrypted files to be stored

Reply:
Added a check https://github.com/projectcaluma/alexandria/blob/e941029fae378f57e5ace86171472b96b5c89956/alexandria/storages/fields.py#L39-L42